examples/extract-auth-token.py

```diff
@@ -4,12 +4,44 @@
 import libflagship.logincache
 import json
 import sys
+import platform
+import os
+import getopt
 
-if len(sys.argv) != 2:
-    print(f"usage: {sys.argv[0]} <path/to/login.json>")
-    exit(1)
+def jankauto():
+    userdir = os.getlogin()
+    useros = platform.system()
 
-data = open(sys.argv[1]).read()
+    darfileloc = f'/Users/{userdir}/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json'
+    winfileloc = os.path.join('C:\\Users\\') + f'{userdir}' + os.path.join('\\AppData\\Local\\Ankermake\\login.json')
+    
+    try:
+        opts, args = getopt.getopt(sys.argv[1:],"haf:", ["help", "auto", "file="])
+    except getopt.GetoptError:
+      print ('extract-auth-token.py -a OR extract-auth-token.py -f <path-to-login.json>')
+      sys.exit(2)
```

Comment:
If you feel up for it, consider using `argparse`, for an easier, more modern argument parsing (see `connect-mqtt.py` for an example)

See the last comment for how to wrap `main()`, then replace `sys.exit(2)` with just `return 2` :)